src/components/Profile/ProfileImage/ProfileImage.tsx

```diff
@@ -1,20 +1,42 @@
 import "./ProfileImage.css";
-import ProfilePicture from "../../../assets/profile.png";
 import clsx from 'clsx';
+import { useEffect, useState } from "react";
 
 type ImageOptions = {
   circular?: boolean;
+  border?: boolean;
 };
 
+
 const ProfileImage = (options: ImageOptions = {}) => {
+  const [ProfilePicture, setProfilePicture] = useState<string | null>(null);

```

Comment:
```suggestion
  const [profilePicture, setProfilePicture] = useState<string | null>(null);

```

Uppercase variables are usually Class objects in JS.